pvt/helpers/src/time.ts

```diff
@@ -8,6 +8,11 @@ export const currentTimestamp = async (): Promise<BigNumber> => {
   return bn(timestamp);
 };
 
+export const currentWeekTimestamp = async (): Promise<BigNumber> => {
+  const { timestamp } = await network.provider.send('eth_getBlockByNumber', ['latest', true]);
+  return bn(timestamp).div(WEEK).mul(WEEK);
```

Comment:
Similarly with this, we're going to have to round up and down timestamps a reasonable amount.

Reply:
This is also going to conflict with a similar change in #1234